### ISSUE TITLE
Crashing when left-clicking config

### ISSUE THREAD

**Issue by Linus42069** [ArnoldasMk/Kali#6]
Whenever I press one of the configs I've saved under the config window, the game crashes.
(debian 10) (what else did I miss to put in here?)

**Comment by Linus42069** [comment]
I've never been able to build other cheats such as MissedIT and Fuzion, neither the newver verisions of Gamesneeze due to not having been able to get gcc and g++ 10 working (as unstable) since the stable verision isn't out for Debian Buster 10 yet. I haven experience like anything as this before but since I have BARLEY even touched a cheat that runs properly before I can't say it only happens when using Kali. Altough when I used Gamesneeze before when I could, I was able to save and load configs. 
But I've been able to save only one config using kali.

Sorry if the message is cluttering, also sorry for a late response
